AUTHORS

```diff
@@ -0,0 +1,26 @@
+Andrew Laski <andrew@lascii.com>
+Balazs Gibizer <balazs.gibizer@ericsson.com>
+Brant Knudson <bknudson@us.ibm.com>
+Clark Boylan <clark.boylan@gmail.com>
+Dan Kenigsberg <danken@redhat.com>
+Edward Betts <edward@4angle.com>
+Francesco Banconi <francesco.banconi@canonical.com>
+Free Ekanayaka <free.ekanayaka@canonical.com>
```

Comment:
Perhaps drop the duplicates?

Reply:
Good catch! The list was autogenerated by pbr it seems, so I did not double check that one. I removed the duplicates in favor of the authors's personal addresses, as these folks do not work for Canonical any more.